contrib/vectorscan/template.py

```diff
@@ -0,0 +1,40 @@
+pkgname = "vectorscan"
+pkgver = "5.4.11"
+pkgrel = 0
+archs = ["aarch64", "ppc64", "x86_64"]
```

Comment:
should probably be ppc64le

(does it not build at all on other platforms?)

Reply:
at the minute no, but I think there will be a software backend in the next release for arch’s with unsupported simd extensions